pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -47,11 +47,10 @@ def __init__(
         checkpoint_io: Optional[CheckpointIO] = None,
         precision_plugin: Optional[PrecisionPlugin] = None,
     ) -> None:
-        self._accelerator = accelerator
+        self.accelerator = accelerator
```

Comment:
why are these renamed? You can't have a property and attribute with the same name. What is the actual fix?

Reply:
I was confused about this PR and thought it changes logic too, but seems not. Only logic change is for checkpoint_io.
Before accelerator and precision_plugin in __init__ set self._accelerator/_precision_plugin directly, with changes in this PR it calling setter functions (`self.accelerator` is actually set `self._accelerator` in the accelerator setter method).

Reply:
Exactly, also see the changed test